terraform/members.tf

```diff
@@ -0,0 +1,171 @@
+resource "github_membership" "amenowanna" {
```

Comment:
We can create a variable for this:
```
variable "admins" {
  type = list(string)
  value = [
    "amenowanna",
    "benr", 
     ...
  ]
}
```
(I may have gotten the type wrong here, you probably don't need the type.  Variables definitions should be kept in `variables.tf`.)

And then:

```
resource "github_membership" "admin" {
    count = len(var.admin)
    username = var.admin[count.index]
    role = "admin"
}
```

There is also a for/each way to do this, but I am not as familiar with that.

Reply:
No, please! :D Do not ask me to make this dynamic now... it is already a pain to sync all the resources with the local state. I will leave the prettification of this for somebody else or at least for phase 2 when we will reorder all teams and members. Does it sound good? 

Pleasee!!